lib/puppet/type/nexus3_repository.rb

```diff
@@ -55,7 +55,7 @@
     cleanup_policies: {
       type: 'Array[String]',
       desc: 'A list of cleanup policies to apply to this repository',
-      default: [],
```

Comment:
If for some reason the server is reporting the array as having one item that is an empty string, the correct is to changer the read script, not to change the default value to this.